File: 4.rb
#сумата на n числа
puts 'Enter n: '
n = gets.chomp.to_i
i = 1
sum = 0
while i<=n
    puts 'Enter a: '
    a = gets.chomp.to_i
    sum = sum + a
    i = i + 1
end
puts "sum = #{sum}"

File: 4.c
#include <stdio.h>
void main()
{
    int n, sum, i, a;
    do{
        printf("Enter n: ");
        scanf("%d", &n);
    }while(n<0);
    sum = 0;
    i = 1;
    while(i<=n){
        printf("Enter a: ");
        scanf("%d", &a);
        sum = sum + a;
        i++;
    }
    printf("sum = %d", sum);
}